<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MstDialog</class>
 <widget class="QDialog" name="MstDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="framePanel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>550</width>
     <height>440</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>511</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>,white</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 26px, white;
border: 0px;</string>
   </property>
   <property name="text">
    <string>主监控列表</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableView" name="tableViewMstList">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>90</y>
     <width>480</width>
     <height>345</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="gridStyle">
    <enum>Qt::NoPen</enum>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonMstReturn">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>380</y>
     <width>170</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonMstHangUp">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>320</y>
     <width>170</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>挂断</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonMstPrev">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>200</y>
     <width>170</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>上一页</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonMstNext">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>260</y>
     <width>170</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>下一页</string>
   </property>
  </widget>
  <zorder>framePanel</zorder>
  <zorder>label_2</zorder>
  <zorder>pushButtonMstReturn</zorder>
  <zorder>pushButtonMstHangUp</zorder>
  <zorder>pushButtonMstPrev</zorder>
  <zorder>pushButtonMstNext</zorder>
  <zorder>tableViewMstList</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CommonPushButton</class>
   <extends>QPushButton</extends>
   <header>commonpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButtonMstPrev</tabstop>
  <tabstop>pushButtonMstNext</tabstop>
  <tabstop>pushButtonMstHangUp</tabstop>
  <tabstop>pushButtonMstReturn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
